helpers/training/save_hooks.py

```diff
@@ -123,6 +130,18 @@ def __init__(
             elif self.args.pixart_sigma:
                 self.ema_model_cls = PixArtTransformer2DModel
 
+        self.pipeline_class = None
+        if self.args.sd3:
+            self.pipeline_class = PIPELINE_MAPPING["sd3"]
+        elif self.args.legacy:
+            self.pipeline_class = PIPELINE_MAPPING["legacy"]
+        elif self.flux:
+            self.pipeline_class = PIPELINE_MAPPING["flux"]
+        else:
+            self.pipeline_class = PIPELINE_MAPPING["sdxl"]
```

Comment:
hmm i guess we're relying on the sdxl pipeline for kolors but what about pixart?

Reply:
oh, right - pixart doesn't do loras